NitroxLauncher/MainWindow.xaml

```diff
@@ -150,72 +42,114 @@
 
-                <Image Margin="32,32,131,483" Source="nitroxLogo.png" Stretch="Fill"/>
-
-                <StackPanel Margin="32,82,0,286" x:Name="SideBarPanel" HorizontalAlignment="Left" Width="196">
+                <StackPanel Margin="32,108,0,0" HorizontalAlignment="Left" Width="196" VerticalAlignment="Top">
+                    <TextBlock HorizontalAlignment="Left" FontSize="10" Foreground="#7FFFFFFF" Margin="0,0,0,4" FontWeight="Bold"><Run Text="PLAY"/></TextBlock>
                     <Button Click="ButtonNavigation_Click" Tag="{StaticResource LaunchGamePage}" ToolTip="Play the game" BorderBrush="#00000000" >
                         <StackPanel Orientation="Horizontal" VerticalAlignment="Center">
-                            <Image Source="{DynamicResource SideBarPlayIcon}" Width="24px" Height="24px" />
+                            <Image Source="{StaticResource SideBarPlayIcon}" Width="24px" Height="24px" />
                             <TextBlock FontSize="16" Margin="12,0,0,0" VerticalAlignment="Center">Play game</TextBlock>
                         </StackPanel>
                     </Button>
-                    <Button Click="ButtonNavigation_Click" Tag="{StaticResource ServerPage}" ToolTip="Configure server options" BorderBrush="#00000000">
+                    
+                    <Button Click="ButtonNavigation_Click" Tag="{StaticResource ServerPage}" ToolTip="Configure and start the server" BorderBrush="#00000000">
                         <StackPanel Orientation="Horizontal" VerticalAlignment="Center">
-                            <Image Source="{DynamicResource SideBarServerIcon}" Width="24px" Height="24px" />
+                            <Image Source="{StaticResource SideBarServerIcon}" Width="24px" Height="24px" />
                             <TextBlock FontSize="16" Margin="12,0,0,0" VerticalAlignment="Center">Server</TextBlock>
                         </StackPanel>
                     </Button>
-                    <Button Click="ButtonNavigation_Click" Tag="{StaticResource OptionPage}" ToolTip="Launcher settings" BorderBrush="#00000000">
+                    
+                    <!--
+                    <Button Click="ButtonNavigation_Click" Tag="{StaticResource LibraryPage}" ToolTip="Subnautica installations and recent servers" BorderBrush="#00000000">
                         <StackPanel Orientation="Horizontal" VerticalAlignment="Center">
-                            <Image Source="{DynamicResource SideBarOptionsIcon}" Width="24px" Height="24px" />
-                            <TextBlock FontSize="16" Margin="12,0,0,0" VerticalAlignment="Center">Settings</TextBlock>
+                            <Image Source="{DynamicResource SidebarLibraryIcon}" Width="24px" Height="24px" />
+                            <TextBlock FontSize="16" Margin="12,0,0,0" VerticalAlignment="Center">Library</TextBlock>
+                        </StackPanel>
+                    </Button>
+                    -->
```

Comment:
If we arent going to have this yet do we want to remove it rather than comment it out? we can always add it in later when the functionality is written